public/video-ui/src/components/ReactApp.js

```diff
@@ -14,6 +14,7 @@ class ReactApp extends React.Component {
   componentWillMount() {
     if (this.props.params.id) {
       this.props.appActions.getVideo(this.props.params.id);
+      this.props.appActions.getPublishedVideo(this.props.params.id);
```

Comment:
Do we need to have two method calls (and two redux actions) for this? Could we combine them into a single one, even if it ends up making two requests to the server?